### DISCUSSION TITLE
Button not re-enabling after AsyncRelayCommand finished.

### DISCUSSION THREAD

**Discussion by Orgbrat** [CommunityToolkit/dotnet#193] [Q&A] [answered]
I am using CommunityToolkit.Mvvm 8.0.0-preview2 in my Maui application. My issue is the Buttons  attached to their AsyncRelayCommands disable when the command starts but they never re-enable when the command is finished. Is there something I am doing wrong or is there still a bug that was not fixed with the [#111](https://github.com/CommunityToolkit/dotnet/pull/111) issue?

```
SyncNowProviderCommand = new AsyncRelayCommand<string>(ExecuteSyncNowProviderCommand);

async Task ExecuteSyncNowProviderCommand(string isAutoSync){ }
```
Orgbrat

**Comment by Sergio0694** [comment, marked as answer]
The bug has been fixed by that PR, but the fix hasn't yet been published 🙂
It will be included in Preview3. If you want, you can try out a preview build from our preview feed though:

```
https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-MainLatest/nuget/v3/index.json
```

**Reply by Orgbrat** [reply]
Approximately when will Preview3 be published? And thanks for the quick reply...

**Reply by Sergio0694** [reply]
There isn't an exact ETA, but I'm aiming for early next week 😊
